app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/OutcomeValueFunction.scala

```diff
@@ -12,6 +13,12 @@ case class OutcomeValueFunction(points: Vector[OutcomeValuePoint]) {
           },
           s"Points must be ascending: $points")
 
+  def toTLV: PayoutFunctionV0TLV = {
+    PayoutFunctionV0TLV(points.map { point =>
+      TLVPoint(point.outcome.toLongExact, point.value, point.isEndpoint)
```

Comment:
Won't toLongExact cause an exception if it's a decimal here?

Reply:
It will and should